### PR TITLE
tests.yml: add paths-ignore

### PR DESCRIPTION
**Pull request by hyuraku**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew typecheck` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----
add `paths-ignore` to `workflow/tests.yml` because I guess `ci` isn't necessary when we open PR for `README.md`.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
We do actually run `CI / syntax` on there. If this could be limited to the other jobs: that'd be best!

**Comment by MikeMcQuaid** [comment]
If it's easier: I also think it'd make more sense to make the other jobs depend on `CI / syntax` and only run when it's complete (as it only takes ~1m).

**Comment by hyuraku** [comment]
> If it's easier: I also think it'd make more sense to make the other jobs depend on `CI / syntax` and only run when it's complete (as it only takes ~1m).

@MikeMcQuaid 
What you said means https://github.com/Homebrew/brew/pull/11236 ?

**Review by MikeMcQuaid — APPROVED** [review]
Thanks again @hyuraku!

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ on:
     branches:
       - master
   pull_request:
+    paths-ignore:
+    - README.md
```
Comment (MikeMcQuaid) [inline .github/workflows/tests.yml:8]: Actually, no, this will fail because the CI job will still be marked as `required` but just won't be run 😢

**Comment by MikeMcQuaid** [comment]
Sorry @hyuraku, don't think we can make this work given our GitHub CI configuration!

**Pull request closed without merging**